hw/snitch_cluster/src/snitch_cluster_wrapper.sv.tpl

```diff
@@ -469,50 +521,93 @@ for i in range(len(cfg['cores'])):
     .tcdm_req_t (${cfg['pkg_name']}::tcdm_req_t),
     .tcdm_rsp_t (${cfg['pkg_name']}::tcdm_rsp_t)
   ) i_cluster (
-    .clk_i,
-    .rst_ni,
+    //-----------------------------
+    // Clock and reset
+    //-----------------------------
+    .clk_i  ( clk_i  ),
+    .rst_ni ( rst_ni ),
+    //-----------------------------
+    // Interrupt ports
+    //----------------------------
 % if cfg['enable_debug']:
-    .debug_req_i,
+    .debug_req_i ( debug_req_i ),
 % else:
     .debug_req_i ('0),
 % endif
-    .meip_i,
-    .mtip_i,
-    .msip_i,
+    .meip_i ( meip_i ),
+    .mtip_i ( mtip_i ),
+    .msip_i ( msip_i ),
+    //-----------------------------
+    // Cluster base addressing
+    //-----------------------------
 % if cfg['tie_ports']:
-    .hart_base_id_i (${to_sv_hex(cfg['cluster_base_hartid'], 10)}),
-    .cluster_base_addr_i (${to_sv_hex(cfg['cluster_base_addr'], cfg['addr_width'])}),
+    .hart_base_id_i ( ${to_sv_hex(cfg['cluster_base_hartid'], 10)} ),
+    .cluster_base_addr_i ( ${to_sv_hex(cfg['cluster_base_addr'], cfg['addr_width'])} ),
 % else:
-    .hart_base_id_i,
-    .cluster_base_addr_i,
+    .hart_base_id_i ( hart_base_id_i ),
+    .cluster_base_addr_i ( cluster_base_addr_i ),
 % endif
+    //-----------------------------
+    // ISO crossings
+    //-----------------------------
 % if cfg['timing']['iso_crossings']:
-    .clk_d2_bypass_i,
+    .clk_d2_bypass_i ( clk_d2_bypass_i ),
 % else:
-    .clk_d2_bypass_i (1'b0),
+    .clk_d2_bypass_i ( 1'b0 ),
 % endif
-    .snax_req_o (snax_req),
-    .snax_qvalid_o (snax_qvalid),
-    .snax_qready_i (snax_qready),
-    .snax_resp_i (snax_resp),
-    .snax_pvalid_i (snax_pvalid),
-    .snax_pready_o (snax_pready),
-    .snax_tcdm_req_i (snax_tcdm_req),
-    .snax_tcdm_rsp_o (snax_tcdm_rsp),
+    //-----------------------------
+    // SNAX Custom Instruction Ports
+    //-----------------------------
+    // Request custom instruction format ports
+    .snax_req_o    ( snax_req    ),
+    .snax_qvalid_o ( snax_qvalid ),
+    .snax_qready_i ( snax_qready ),
+    // Response custom instruction format ports
+    .snax_resp_i   ( snax_resp   ),
+    .snax_pvalid_i ( snax_pvalid ),
+    .snax_pready_o ( snax_pready ),
+    //-----------------------------
+    // SNAX CSR Ports
+    //-----------------------------
+    // Request for CSR format ports
+    .snax_csr_req_bits_data_o   ( snax_csr_req_data  ),
+    .snax_csr_req_bits_addr_o   ( snax_csr_req_addr  ),
+    .snax_csr_req_bits_write_o  ( snax_csr_req_write ),
+    .snax_csr_req_valid_o       ( snax_csr_req_valid ),
+    .snax_csr_req_ready_i       ( snax_csr_req_ready ),
+    // Response for CSR format ports
+    .snax_csr_rsp_bits_data_i   ( snax_csr_rsp_data  ),
+    .snax_csr_rsp_valid_i       ( snax_csr_rsp_valid ),
+    .snax_csr_rsp_ready_o       ( snax_csr_rsp_ready ),
+
```

Comment:
We leave these two configuration ports first for the ci passing and will fix it later!

Reply:
Yes! So just for documentation purposes let's first fix the GeMM, SIMD and reshuffle imports then fix it accordingly 😄